src/branding-bar/BrandingBar.js

```diff
@@ -1,8 +1,62 @@
 import React from "react";
-import styled from "styled-components";
+import styled, { css } from "styled-components";
 
-const BrandingBarContainer = styled.header``;
+import LogoIconSrc from "../assets/icons/recidiviz_logo.svg";
+import { LinkPill } from "../pill";
+
+const brandingBarFlexProperties = css`
+  align-items: center;
+  display: flex;
+`;
+
+const BrandingBarContainer = styled.header`
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 50px;
```

Comment:
Honestly, I just hard-coded this value to eyeball match the mocks (it's actually still too small).  At any rate, what would you think about me adding a `spacing` section to the theme to allow us to use consistent spacing throughout the app?  Something like what [Tailwind CSS does](https://github.com/tailwindcss/tailwindcss/blob/master/stubs/defaultConfig.stub.js#L132-L152).  TBD what the actual values would be.

Reply:
I've added a proposed solution in the [latest commit](https://github.com/Recidiviz/public-dashboard/pull/22).  If we don't link it, I can just yank the commit.

Reply:
And... the correct link:  https://github.com/Recidiviz/public-dashboard/pull/20/commits/c3f3ff0f83880e8e20b515e9101034fa1d6d0ffd

Reply:
i am not a huge fan of this approach honestly ... i kind of just philosophically don't actually believe that "consistent spacing" is a thing? In practice I found this feature of Tailwind to be more trouble than it was worth on the COVID site ... the sizing numbers correspond to somewhat arbitrary rem values so I would mostly find myself choosing spacing numbers at semi-random and eyeballing until it looked OK. I don't really see how that's any better than eyeballing pixel values or getting them from Figma

Reply:
No worries, I can revert the commit.  I'll still attempt to use semi-consistent spacing values, but we'll see how long that holds up and how well the design is supported (or not supported) in doing so.